src/app/kbv/controllers/load-question.js

```diff
@@ -26,12 +26,12 @@ class LoadQuestionController extends BaseController {
       next();
     });
   }
-  next(req) {
+  hasQuestion(req) {
     if (req.session?.question) {
-      return "question";
+      return true;
     }
 
-    return "done";
+    return false;
```

Comment:
This can be simplified to `return !!req.session?.question`

Reply:
It might be a personal preference but I find the shorter syntax easier to understand. It's a commonly used idiom in JS/TS so I see no issue with it personally.

Reply:
If opional chaning returns `undefined`, then `!!` will evaluate to false as well.